doc/exts/pylint_messages.py

```diff
@@ -23,6 +23,8 @@
 PYLINT_MESSAGES_PATH = PYLINT_BASE_PATH / "doc" / "messages"
 """Path to the messages documentation folder."""
 
+PYLINT_MESSAGES_DATA_PATH = PYLINT_BASE_PATH / "doc" / "data" / "messages"
+"""Path to the messages documentation folder."""
```

Comment:
Can we distinguish these two docstrings?

Reply:
Oops, that's a mistake. Fixing!